contrib/changelog.hbs

```diff
@@ -1,4 +1,4 @@
-{{header}}
+{{header}} - {{date}}
 {{toHeader header "-"}}
```

Comment:
The toHeader must receive the same above text, so that they have the same width

Reply:
Do i need to re submit the code?

Reply:
Just do the requested changes on this same branch and do `push` again, GitHub will update the PR automatically :). Let me know if you need more help.